lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -166,24 +166,49 @@ def custom_handler(signum, frame):
 
 
 @pytest.mark.parametrize(
-    'qt_key, qt_mods, answer',
+    "qt_key, qt_mods, answer",
     [
-        ('Key_A', ['ShiftModifier'], 'A'),
-        ('Key_A', [], 'a'),
-        ('Key_A', ['ControlModifier'], 'ctrl+a'),
-        ('Key_Aacute', ['ShiftModifier'],
-         '\N{LATIN CAPITAL LETTER A WITH ACUTE}'),
-        ('Key_Aacute', [],
-         '\N{LATIN SMALL LETTER A WITH ACUTE}'),
-        ('Key_Control', ['AltModifier'], 'alt+control'),
-        ('Key_Alt', ['ControlModifier'], 'ctrl+alt'),
-        ('Key_Aacute', ['ControlModifier', 'AltModifier', 'MetaModifier'],
-         'ctrl+alt+meta+\N{LATIN SMALL LETTER A WITH ACUTE}'),
+        ("Key_A", ["ShiftModifier"], "A"),
+        ("Key_A", [], "a"),
+        (
+            "Key_A",
+            ["ControlModifier"],
+            ("ctrl+a" if sys.platform != "darwin" else "cmd+a"),
```

Comment:
I find the previous version a little easier to read with more on a single line. Perhaps you could leave the parametrize section alone and put the switch inside the function?
```python
if sys.platform == "darwin":
    qt_mods.replace("ctrl", "cmd")
```

Reply:
fair, that did not occur to me to try! I was going going through the failing tests one at a time and putting the fix in.

On the other hand, changing the parameters makes it clear from looking at the test the expected behavior is which might be clearer in the long run?